### DISCUSSION TITLE
Grief Prevention -> Luck Perms / Ranks

### DISCUSSION THREAD

**Discussion by Reiamsei** [GriefPrevention/GriefPrevention#1208] [Ideas]
My Problem:
I want to give diffrent ranks a diffrent amount of InitialBlocks and a diffrent amount of Claims.
Default : 
InitialBlocks: 1000
MaximumNumberOfClaimsPerPlayer: 1
Premium : 
InitialBlocks: 2000
MaximumNumberOfClaimsPerPlayer: 2
....

so that, when I give Player the rank Premium, he gets 2000 Blocks to claim, and when he gets Default back he only has 1000 Blocks again.
The same with the max numver of claims.

I tried to use /acb , but this is not working and dont make any sence when you have more then 2 ranks in my opinion...

### Server and GriefPrevention version:
Server Version : 1.16.4
Grief Prevention version : 16.16.0
Luckperms version : 5.2.76

would be awesome, if someone can help me !
sorry for my bad english

**Comment by RoboMWM** [comment]
this is not a bug, this is an addon suggestion. Moving to discussions.

**Comment by hafarooki** [comment]
Is there any way to do this as of now, either via config or API?

**Reply by Jikoo** [reply]
Theoretically it's possible to have per-rank starting claim block amounts with base GP. There's an undocumented feature of applying certain commands to permissions specified `[permission.node]` including brackets. That's supported by `/acb` and others, so you could set up nodes with different base blocks (though that's internally very messy and I don't know if permission-based blocks will remain in existence in future GP versions). That said, I believe GPAccruals already offers per-group starting blocks, so that would probably be easier for an admin to set up.

I am not aware of an addon that enacts per-group claim limits, but it would certainly be possible for someone to create one that fulfils both requested features using GP's API.
